notes-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -1312,4 +1325,17 @@ private Page updateChildrenContainer(Page note) throws WikiException {
     return noteService.updateNote(note);
   }
 
+  private String validateGroupWikiOwner(String wikiOwner) {
```

Comment:
There is not validation inside, IMO it's : 
```suggestion
  private String extractGroupWikiOwner(String wikiOwner) {
```
or
```suggestion
  private String formatWikiOwnerToGroupId(String wikiOwner) {
```